atomic_defi_design/Dex/Components/CoinMenu.qml

```diff
@@ -28,7 +28,15 @@ Menu {
         id: disable_action
         text: qsTr("Disable %1", "TICKER").arg(ticker)
         onTriggered: API.app.disable_coins([ticker])
-        enabled: General.canDisable(ticker)
+        enabled:
+        {
+            let progress = General.zhtlcActivationProgress(current_ticker_infos.activation_status, ticker)
```

Comment:
ZHTLC should not be called when the current ticker is not a ZHTLC asset...
Put this statement inside the if below ?

Reply:
Maybe we can put the zhtlc logic inside the canDisable function directly